config/src/config/firehose_streamer_config.rs

```diff
@@ -7,8 +7,11 @@ use serde::{Deserialize, Serialize};
 #[serde(default, deny_unknown_fields)]
 pub struct FirehoseStreamerConfig {
     pub enabled: bool,
-    // The version to start pushing out indexed data from, for the StreamingFast Firehose indexer
-    // Alternatively can set the `STARTING_VERSION` env var
     #[serde(default, skip_serializing_if = "Option::is_none")]
+    #[deprecated = "please use starting_block instead"]
     pub starting_version: Option<u64>,
```

Comment:
You should remove this if it doesn't work at all

Reply:
backward compatibiility?